package.json

```diff
@@ -35,9 +43,15 @@
     "eslint-plugin-prettier": "^3.1.4",
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-sonarjs": "^0.5.0",
-    "italia-tslint-rules": "^1.1.3",
-    "italia-utils": "^6.2.0",
-    "jest": "^25.2.7",
+    "express": "^4.17.1",
+    "faker": "^5.1.0",
+    "fp-ts": "1.12.0",
```

Comment:
I think you can use the fp-ts 1.17.4, or is there some reason to use 1.12.0?

Reply:
in fact we must use `1.17.4` that is the same version used by `io-ts-commons` to avoid having multiple versions (will break the build): https://github.com/pagopa/io-ts-commons/blob/master/package.json#L47

Reply:
No, I simply sticked to the version used in IO-APP. I'll give it a try. Thank you for pointing out